emcc.py

```diff
@@ -59,6 +59,13 @@
 
 logger = logging.getLogger('emcc')
 
+# In git checkouts of emscripten `bootstrap.py` exists to run post-checkout
+# steps.  In packaged versions (e.g. emsdk) this file does not exist
+# and these steps are assumed to have been run already.
+if os.path.exists(utils.path_from_root('bootstrap.py')):
+  import bootstrap
+  bootstrap.check()
```

Comment:
Doesn't this add a lot of work in git checkouts, in each emcc.py invocation?

Reply:
`.check()` simply checks the timestamps of (currently) 2 files.   Its designed to be super cheap.